core/src/main/scala/kafka/log/FileMessageSet.scala

```diff
@@ -128,11 +128,11 @@ class FileMessageSet private[kafka](@volatile var file: File,
 
   /**
    * Search forward for the file position of the last offset that is greater than or equal to the target offset
-   * and return its physical position. If no such offsets are found, return null.
+   * and return its physical position and the size of the message. If no such offsets are found, return null.
```

Comment:
Perhaps "... return its physical position and the size of the message **at that offset**"?